src/Aquifer.Jobs/Subscribers/TranslationSubscriber.cs

```diff
@@ -0,0 +1,379 @@
+                s_durableFunctionTaskOptions);
+        }
+        catch (Exception orchestrationException)
+        {
+            // All the Activities orchestrated here have a retry policy.  If things are still failing after multiple retries
+            // then something is very wrong. If this happens then swallow errors here to allow successful orchestration function
+            // completion but publish a poison queue item for manual retry later.
+            var ct = CancellationToken.None;
+
+            try
+            {
+                _logger.LogError(
+                    orchestrationException,
+                    "Error translating resource content for Project ID {ProjectId}. A poison message will be published to \"{PoisonQueueName}\" to enable manual retry. Manual dev intervention is required.",
+                    dto.ProjectId,
+                    dto.PoisonQueueName);
+
+                var translateProjectResourcesPoisonQueueClient = await _queueClientFactory.GetQueueClientAsync(
+                    dto.PoisonQueueName,
+                    ct);
+                await translateProjectResourcesPoisonQueueClient.SendMessageAsync(dto.OriginalQueueMessageText, ct);
+            }
+            catch (Exception queuePublishingException)
+            {
+                // don't allow errors during poison queue publishing to fail the durable function
+                _logger.LogError(
+                    queuePublishingException,
+                    "After an error translating resource content for Project ID {ProjectId} another error occurred when attempting to publish a poison message to \"{PoisonQueueName}\". Manual dev intervention is required to replay the message. Message text: {MessageText}",
+                    dto.ProjectId,
+                    dto.PoisonQueueName,
+                    dto.OriginalQueueMessageText);
+            }
+        }
+    }
+
+    [Function(nameof(TranslateProjectResourceActivity))]
+    public async Task<int> TranslateProjectResourceActivity(
+        [ActivityTrigger] TranslateProjectResourceActivityDto dto,
+        FunctionContext activityContext)
+    {
+        await TranslateResourceCoreAsync(
+            dto.ResourceContentId,
+            dto.StartedByUserId,
+            TranslationOrigin.Project,
+            activityContext.CancellationToken);
+
+        return dto.ResourceContentId;
+    }
+
+    [Function(nameof(UpdateProjectPostTranslationActivity))]
+    public async Task UpdateProjectPostTranslationActivity(
+        [ActivityTrigger] UpdateProjectPostTranslationActivityDto dto,
+        FunctionContext activityContext)
+    {
+        _logger.LogInformation(
+            "Beginning post-translation project updates for Project ID {ProjectId}.",
+            dto.ProjectId);
+
+        var project = await _dbContext.Projects
+            .SingleAsync(p => p.Id == dto.ProjectId);
+
+        var companyLeadUserId = project.CompanyLeadUserId
```

Comment:
Looks like this value is never used.

Reply:
Fix: https://github.com/BiblioNexusStudio/aquifer-server/pull/515/commits/e0aedde6f1a5abb26579ff5a1f4687e31f5d47c7